dtests/src/test/java/io/snappydata/hydra/cluster/SnappyPrms.java

```diff
@@ -85,6 +85,13 @@
     public static Long useSplitMode;
 
     /**
+     * (boolean) - whether stop mode needs to be checked before deleting the config data if already exists.
+     * This is required in case user wants to start the cluster and then stop the same later on using different script.
+     * In this case, test should not delete the existing configuration data created by previous test.
+     */
+    public static Long isStopMode;
```

Comment:
why this is needed? In any condition test should not delete any configuration data of old  test.